lib/business_time/version.rb

```diff
@@ -1,3 +1,3 @@
 module BusinessTime
-  VERSION = "0.8.1"
+  VERSION = "0.8.2a"
```

Comment:
This is required for some project where we actually need packaged gems, I'll change this, thanks.